packages/clay-core/src/overlay-mask/OverlayMask.tsx

```diff
@@ -0,0 +1,218 @@
+/**
+ * SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {TInternalStateOnChange, useInternalState} from '@clayui/shared';
+import React, {useEffect, useLayoutEffect, useRef, useState} from 'react';
+
+type Bounds = {
+	height: number;
+	width: number;
+	x: number;
+	y: number;
+};
+
+const M = (x: number, y: number) => `M ${x} ${y}`;
```

Comment:
What is M and L?

Machine Learning or Mercado Livre 😂 I'm joking but I would understand what is behind it

Reply:
😂  just a shortcut to not replicate but you are right I think I should add the most exact word for it